scripts/can_bundle.py

```diff
@@ -88,9 +87,11 @@ def fetch_bundle(source: str, bundle_uuid: str, bundle_version: str) -> Bundle:
             for plugin_source_spec in plugin.sources:
                 if source_ref.spec.contains(plugin_source_spec):
                     plugin_source_ref = plugin.resolve_source(str(plugin_source_spec))
-                    fqid = SourcedBundleFQID(source=plugin_source_ref,
-                                             uuid=bundle_uuid,
-                                             version=bundle_version)
+                    fqid = plugin.resolve_bundle(dict(
+                        source=plugin_source_ref.to_json(),
```

Comment:
```suggestion
                    fqid = plugin.resolve_bundle(dict(source=plugin_source_ref.to_json(),
```

… no that you use dict